### ISSUE TITLE
Manage project delegates

### ISSUE THREAD

**Issue by erak** [radicle-dev/radicle-cli#118]
### Description

Implement `rad delegate` with the following commands: 

- `rad delegate`: alias for `rad delegate list`
- `rad delegate list`: list delegates of current project
- `rad delegate add <peer-id>`: adds delegate to current project, do not re-add own peer id
- `rad delegate remove <peer-id>`: removes peer from delegates, do not remove own peer id

**Comment by cloudhead** [comment]
Yeah, though actually we should probably start by supporting indirect delegations (user URNs) before peer-ids, as that's what all projects use at the moment.
